<!--
  arbe_phoenix_radar_driver Launch 
 -->
<launch>
    <arg name="video_device_num" default="0" />
    <arg name="antenna_num" default="0" />
    <node name="cv_camera_$(arg antenna_num)" pkg="cv_camera" type="cv_camera_node" output="screen" required="false">
        <param name="image_width" value="1920" />
        <param name="image_height" value="1080" />
        <param name="device_id" value="$(arg video_device_num)" />
    </node>
</launch>

